exercises/G-numbers/exercise.js

```diff
@@ -1 +1,10 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+
+   let numberOfStudents = 15;
+   let numberOfMentors = 8;
+
+   let totalNum = numberOfStudents + numberOfMentors;
+
+   console.log("Number of students: "+ numberOfStudents);
```

Comment:
Hi Howard, since you are calculating the total sum of students and mentors in the totalNum variable;
console logging that and concatenate a string before the total should do the trick